### DISCUSSION TITLE
blog/2024/01/i-broke-my-computer...twice/

### DISCUSSION THREAD

**Discussion by giscus** [userliluzibert/userliluzibert.github.io#6] [Giscus]
# blog/2024/01/i-broke-my-computer...twice/

image from IT Crowd S1E5
I already messaged this to my friend already, so to keep myself from coming up with a new way to reformat those words, I’ll just paste what I messaged them here.
I don’t have this blog listed on my resume or anything, so I feel like it would be kind of safe to curse on here. Albeit, if an employer is going through every repo and sees this they might be a little put off.

https://liluzibird.github.io/blog/2024/01/i-broke-my-computer...twice/

**Comment by liluzibird** [comment]
Test message

Friend said comment section not working.
